aten/src/ATen/detail/AcceleratorHooksInterface.h

```diff
@@ -20,6 +20,23 @@ struct TORCH_API AcceleratorHooksInterface {
   // squelch -Werror=non-virtual-dtor
   virtual ~AcceleratorHooksInterface() = default;
 
+  // Whether this backend was enabled at compilation time.
+  // This function should NEVER throw.
+  virtual bool isBuilt() const {
+    return false;
+  }
+
+  // Wether this backend can be used at runtime, meaning it was built,
+  // runtime dependencies are available (driver) and at least one
+  // supported device can be used.
+  // This function should NEVER throw and NOT initialize the context
+  // on any device (result of hasPrimaryContext below should not change).
```

Comment:
```suggestion
  // This function should NEVER throw. This function should NOT initialize the context
  // on any device (result of hasPrimaryContext below should not change).
```

this is confusing cuz of the double negative. I understand the function should NEVER throw.
but should it NEVER NOT initialize the context? Or should it NOT initialize the context on any device?